api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -64,13 +64,18 @@ type VirtualMachineImageSpec struct {
 
 // VirtualMachineImageStatus defines the observed state of VirtualMachineImage
 type VirtualMachineImageStatus struct {
+	// Deprecated
 	Uuid          string `json:"uuid,omitempty"`
+
+	// Deprecated
 	InternalId    string `json:"internalId"`
+
+	// Deprecated
 	PowerState    string `json:"powerState,omitempty"`
 
-	// GOSCSupported describes whether the VirtualMachineImage is GuestOS customizable based on
-	// supported OSTypes for the cluster
-	GOSCSupported *bool  `json:"goscSupported,omitempty"`
+	// GuestOSCustomizable indicates whether the VirtualMachineImage's osType is supported for
+	// customisation on the cluster
+	GuestOSCustomizable *bool  `json:"goscSupported,omitempty"`
```

Comment:
The json marker needs to be updated as well

Reply:
ah! Good catch